python/ray/dag/context.py

```diff
@@ -25,7 +25,7 @@ class DAGContext:
     Args:
         buffer_size_bytes: The maximum size of messages that can be passed
             between tasks in the DAG.
-        async_max_queue_size: The max queue size for the async execution.
+        max_async_queue_size: The max queue size for the async execution.
```

Comment:
Specify that this only applies if `enable_asyncio=True` is used.

Reply:
Updated https://github.com/ray-project/ray/pull/46268/commits/2c916f9f1c30e9dd94e3bd2aa4647a54de3b7e21